openstack/elb/v3/l7policies/results.go

```diff
@@ -23,6 +23,9 @@ type L7Policy struct {
 	// The position of this policy on the listener.
 	Position int32 `json:"position"`
 
+	// The priority of this policy on the listener.
+	Priority int32 `json:"priority,omitempty"`
```

Comment:
omitempty only used to build the request body, in API response parsing progress, this lable is useless.